### PR TITLE
[BUG] Resolve the compilation error of conv3x3.s

### LINKED ISSUE #1374: [Kernel] Compilation error of conv3x3.s
I am using develop of miopen, the command line as follows:
```
MIOpenDriver conv -n 2 -c 64 -H 192 -W 288 -k 128 -y 3 -x 3 -p 1 -q 1 -u 1 -v 1 -l 1 -j 1 -m conv -g 2 -F 1 -t 1 -V 0
```
Then I get errors list in the picture:
image

The following lines in conv3x3.s seems to cause the error. 
image

Then I modify the file, the error is eliminated:
image

### PR DESCRIPTION
**Pull request by zpwenjh**
fixes #1374 
Please refer to #1374  for details of this pr.

### CODE REVIEW

**Comment by carlushuang** [comment]
This fix make sense since otherwise the macro syntax was error.
cc @DrizztDoUrden for confirm. I guess this branch never go in before?

**Comment by atamazov** [comment]
@Slimakanzer @Kirpich30000 Please review this, thanks.

**Review by junliume — APPROVED** [review]
Ping @Slimakanzer @Kirpich30000 @asroy @zjing14

**Review by Kirpich30000 — DISMISSED** [review]
LGTM

**Review by Slimakanzer — COMMENTED** [review]
@zpwenjh  Thank you!
AFACS there is another issue only with gfx90a arch, let me take some time to make changes.

File: src/kernels/conv3x3.s [hunk 1]
```diff
@@ -823,7 +824,7 @@ loop_begin:
   .load_input linesB, mbufs_cnt_B, linesB_start_offset
   .move_input_ptr 1
 
-  s_waitcnt vmcnt(1*mbufs_cnt_A) & lgkmcnt(0)
```
Comment (Kirpich30000) [inline src/kernels/conv3x3.s:826]: This change doesn't look right. At this point we want to make sure that all linesA are ready and we are ok to have linesB still inflight. Isn't it?

Comment (Kirpich30000) [reply src/kernels/conv3x3.s:826]: Sorry, I looked at the old version. The new one is correct.

**Review by Kirpich30000 — APPROVED** [review]
LGTM

**Review by junliume — APPROVED** [review]
Ping @carlushuang

**Pull request merged**